framework/src/consumption/examples/redshift-serverless-workgroup-bootstrap.lit.ts

```diff
@@ -0,0 +1,37 @@
+import { App, Stack } from "aws-cdk-lib";
+import { Construct } from "constructs";
+import * as dsf from '../../index';
+import { Bucket } from "aws-cdk-lib/aws-s3";
+import { ManagedPolicy, Role, ServicePrincipal } from "aws-cdk-lib/aws-iam";
+
+/// !show
+class ExampleRedshiftServerlessWorkgroupBootstrapStack extends Stack {
+    constructor(scope: Construct, id: string) {
+        super(scope, id)
+
+        const bucket = new Bucket(this, "ExampleDataBucket")
+
+        const ingestionRole = new Role(this, "IngestionRole", {
+            assumedBy: new ServicePrincipal("redshift.amazonaws.com"),
+            managedPolicies: [
+                ManagedPolicy.fromAwsManagedPolicyName("AmazonRedshiftAllCommandsFullAccess")
+            ]
+        })
+
+        bucket.grantRead(ingestionRole)
+
+        const workgroup = new dsf.consumption.RedshiftServerlessWorkgroup(this, "DefaultRedshiftServerlessWorkgroup", {
+            workgroupName: "default",
+            defaultNamespaceDefaultIAMRole: ingestionRole
+        })
+
+        const dataAccess = workgroup.accessData(true)
+        const createTable = dataAccess.runCustomSQL("defaultdb", "CREATE TABLE customer(customer_id varchar(max), salutation varchar(max), first_name varchar(max), last_name varchar(max), birth_country varchar(max), email_address varchar(max), address_id varchar(max), gender varchar(max), marital_status varchar(max), education_status varchar(max), purchase_estimate bigint, credit_rating varchar(max), buy_potential varchar(max), vehicle_count bigint, lower_bound bigint, upper_bound bigint, birth_date varchar(max), customer_datetime varchar(max)) diststyle even", "drop table customer")
```

Comment:
Would it be possible to reduce the size of SQL statement here? Just to make it easily readable in the website